pandas/tests/apply/test_series_apply.py

```diff
@@ -795,10 +818,19 @@ def test_apply_to_timedelta():
 @pytest.mark.parametrize("how", ["agg", "apply"])
 def test_apply_listlike_reducer(string_series, ops, names, how):
     # GH 39140
-    expected = Series({name: op(string_series) for name, op in zip(names, ops)})
-    expected.name = "series"
     result = getattr(string_series, how)(ops)
-    tm.assert_series_equal(result, expected)
+    if get_option("new_udf_methods"):
+        if how == "apply":
+            expected = DataFrame({name: string_series for name, op in zip(names, ops)})
+        else:
+            expected = Series(
+                {name: op(string_series) for name, op in zip(names, ops)}, name="series"
+            )
```

Comment:
Series.apply with a list of ops now acts in the same way as Series.apply with a single op, applying the UDF to each row of the Series. This is a no-op for sum/mean.